.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: foreign_travel_content_checker_test
+        ports:
+          - 5432:5432
+        options:
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby and install gems
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Setup the database
+        env:
+          RAILS_ENV: test
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: foreign_travel_content_checker_test
```

Comment:
It'd be more common to set this up through a DATABASE_URL environment variable. 

Typically for testing apps we've used TEST_DATABASE_URL e.g: https://github.com/alphagov/govuk-docker/blob/main/projects/content-publisher/docker-compose.yml#L31-L33

Reply:
Thanks - removed

Reply:
When I tried to remove these, I got a `postgres service is unhealthy` error. Found this advice: https://github.com/jveillet/tentacles-dashboard/issues/205, then this https://github.community/t/mysql-service-never-comes-up-healthy-in-action/16890 which suggests that something has changed with GitHub actions and DRYing it no longer works 🤷